pkg/workflow/argo.go

```diff
@@ -33,10 +34,18 @@ const (
 )
 
 var (
-	timeout            int64 = 3600
-	defaultParallelism int64 = 50
+	timeout            Timeout = "30m"
+	defaultParallelism int64   = 50
 )
 
+type Timeout string
```

Comment:
@jonathan-innis Since I need a `*int64` I have to use a custom type. Can you think of another way of doing this?